articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+---
+title: UKCloud for Microsoft Azure Service Scope | UKCloud Ltd
+description: Outlines important details regarding UKCloud for Microsoft Azure
+services: Azure
+author: Andy Webb
+toc_rootlink: Service Scope
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for Microsoft Azure Service Scope
+toc_fullpath: Service Scope/asz-sco.md
+toc_mdlink: asz-sco.md
+---
+
+# UKCloud for Microsoft Azure Service Scope
+
+## About this document
+
+This document describes the boundaries of the UKCloud for Microsoft Azure service, along with the division of responsibilities between UKCloud and the customer, to facilitate the provisioning and ongoing use of the service.
+
+## About UKCloud for Microsoft Azure
+
+UKCloud for Microsoft Azure harnesses the innovation of Microsoft’s Azure platform and we deliver this service from within our award-winning government-grade multi-cloud environment – addressing the service, connectivity and diversity requirements of UK public sector organisations and their industry partners. The service is based on the Microsoft Azure Stack product, which has been developed by Microsoft as an extension of the Azure public cloud.
+
+## Service options
+
+You can build a tailored solution to meet your requirements by mixing and matching service options. The choice does not lock you to one service or size, allowing you to scale resources up or down as your requirements change.
+
+By default customers will initially have access to:
+
+- XX vCPU
+- YY GB memory
+- ZZ GB storage
+
+The options below are all currently available within the service:
+
+## IaaS Compute 
+Virtual machines (VMs) provide the basic building block providing compute resource within UKCloud for Microsoft Azure. Each standard machine has a pre-allocated number of vCPU's, amount of memory and a temporary disk. The available sizes and families of VM's are contiually increasing, broadly, the following machine families are supported:
+
+- Basic and Standard A-series. Have CPU performance and memory configurations best suited for entry level workloads like development and test.  
```

Comment:
Consistent use of hyphen and uppercase/lowercase s for series (use whichever Microsoft use)